app/components/post/post.js

```diff
@@ -407,6 +413,10 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         replyBarLast: {
             paddingBottom: 10,
         },
+        commentMentionBgColor: {
+            backgroundColor: theme.mentionHighlightBg || 'rgb(255, 255, 0)',
```

Comment:
Why the or here?

Reply:
> Why the or here?

I didn't quite get that.

we are picking the color from theme if exits and defaulting to yellow as we do in webapp.
If you are asking about why even a style. I am 0/5 and just following other how we are adding other condition classes for replyBar.

Reply:
I was asking about the ‘||’ that is an “or” syntax right? Hehehe

Anyway I don’t think we need to default to that and the webapp code probably to old

Reply:
> I was asking about the ‘||’ that is an “or” syntax right?

Aah got it!.